catalyst/utils/tests/test_config.py

```diff
@@ -0,0 +1,37 @@
+# flake8: noqa
+import argparse
+import pytest
+
+from .. import config
+
+
+def test_parse_config_args():
+    configuration = {
+        "stages": {
+            "one": "uno",
+            "two": "dos",
+            "three": "tres"
+        },
+        "key": {
+            "value": "key2"
+        }
+    }
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--command")
+
+    args, uargs = parser.parse_known_args(
+        [
+            "--command", "train", "--path=test.yml:str",
+            "--stages/zero=cero:str"
+        ]
+    )
+
+    configuration, args = config.parse_config_args(
+        config=configuration, args=args, unknown_args=uargs
+    )
+    assert args.command == "train"
```

Comment:
🤔 one another suggestion, to move all `args` arguments to config['args'], maybe?